assets/blocks/course-outline/edit.js

```diff
@@ -85,8 +98,39 @@ const EditCourseOutlineBlock = ( {
 	);
 };
 
-export default withSelect( ( select ) => {
-	return {
-		structure: select( COURSE_STORE ).getStructure(),
+const applyWithSelect = withSelect( ( select ) => ( {
+	structure: select( COURSE_STORE ).getStructure(),
+} ) );
+
+const applyWithDispatch = withDispatch( ( dispatch, ownProps, { select } ) => {
+	const editPost = select( 'core/edit-post' );
+	const { toggleEditorPanelEnabled } = dispatch( 'core/edit-post' );
+
+	const hasLegacyMetaBoxes =
+		editPost.isEditorPanelEnabled( 'meta-box-course-lessons' ) ||
+		editPost.isEditorPanelEnabled( 'meta-box-module_course_mb' );
+
+	const toggleLegacyMetaboxes = ( enable ) => {
+		// Side-effect - Prevent submit modules.
+		document
```

Comment:
It's not a very fancy solution to prevent the modules' saving. I think the ideal solution would be to ignore it on the server, but I debugged the WP core and I didn't find any filter to prevent that. There's a filter to change some posted data (`wp_insert_post_data`), but it doesn't filter the taxonomies.

Reply:
I think it's good. Might have to look out when we do this for the other blocks, if they already have `disabled` inputs, or `select` inputs.